openshift_metrics/invoice.py

```diff
@@ -9,20 +9,23 @@
 GPU_A100 = "NVIDIA-A100-40GB"
 GPU_A100_SXM4 = "NVIDIA-A100-SXM4-40GB"
 GPU_V100 = "Tesla-V100-PCIE-32GB"
+GPU_H100 = "???-H100-???-???" # TODO Confirm name of H100 in Openstack
 GPU_UNKNOWN_TYPE = "GPU_UNKNOWN_TYPE"
 
 # GPU Resource - MIG Geometries
 # A100 Strategies
 MIG_1G_5GB = "nvidia.com/mig-1g.5gb"
 MIG_2G_10GB = "nvidia.com/mig-2g.10gb"
 MIG_3G_20GB = "nvidia.com/mig-3g.20gb"
+# TODO Will we allow MIG on H100s?
 WHOLE_GPU = "nvidia.com/gpu"
 
 # SU Types
 SU_CPU = "OpenShift CPU"
 SU_A100_GPU = "OpenShift GPUA100"
 SU_A100_SXM4_GPU = "OpenShift GPUA100SXM4"
 SU_V100_GPU = "OpenShift GPUV100"
+SU_H100_GPU = "OpenShift GPUH100" # TODO Confirm name of H100 SU Name
```

Comment:
@naved001 I already asked this somewhere. Will remove after confirmation of the SU name

Reply:
That's a question for @joachimweyl

Reply:
We get to name it what we want in the system, so we will go with SU_H100_GPU.

Reply:
`SU_H100_GPU` is the constant name in the python code. The SU name that'll show up reports will be "OpenShift GPUH100"

Reply:
@joachimweyl I assume `"OpenShift GPUH100"` will be fine as the H100 SU name on Openshift.